charts/operator/Chart.yaml

```diff
@@ -6,11 +6,11 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 0.4.26
+version: 0.4.27
 
 # This is the default version of the operator being deployed.
 # ** NOTE for maintainers: please enssure the artifacthub image annotation is updated before merging
-appVersion: v2.1.28-24.2.1
+appVersion: v2.1.29-24.2.1
```

Comment:
Should this be `v2.1.29-24.2.2`?

Reply:
It should but Chris was quicker with the release. I could create a new tag and ask devprod to remove the old image